Source/Evolve/WarpXComputeDt.cpp

```diff
@@ -84,8 +86,10 @@ WarpX::ComputeDt ()
 
 void
 WarpX::PrintDtDxDyDz ()
-{
-    for (int lev=0; lev <= max_level; lev++) {
+{   int Finelev;
+    if (!FinelevInit_flag) Finelev = max_level;
```

Comment:
I would suggest adding a comment to explain why this is needed.

Reply:
Again, I don't think this needs to be modified.  We could also always print from level 0 to the finest level if in InitData we move the call to WarpX::PrintDtDxDyDz after `InitFromScratch` is called.